environments/prod/crime-portal-prod.cer

```diff
@@ -0,0 +1,40 @@
+-----BEGIN CERTIFICATE-----
+MIIHCDCCBfCgAwIBAgITHAAAM9wfaFtNd8fuLwABAAAz3DANBgkqhkiG9w0BAQsF
+ADBMMRgwFgYKCZImiZPyLGQBGRYIaW50ZXJuYWwxFzAVBgoJkiaJk/IsZAEZFgds
+ZXhubGUxMRcwFQYDVQQDEw5FVUNSU0FFTlRDQTAwMTAeFw0yMzEwMTkxMTE3MDla
+Fw0yNDEwMTgxMTE3MDlaMIGHMQswCQYDVQQGEwJHQjEXMBUGA1UECBMOR3JlYXRl
+ciBMb25kb24xDzANBgNVBAcTBkxvbmRvbjEMMAoGA1UEChMDQ0dJMRAwDgYDVQQL
+EwdKdXN0aWNlMS4wLAYDVQQDEyVsbmNzLWNyaW1lcG9ydGFsLW5vdGlmeS11YXQu
+bG5jcy5obWNzMIIBIjANBgkqhkiG9w0BAQEFAAOCAQ8AMIIBCgKCAQEAzsvzZgzB
+Haf6lZ83HmoGZUtd/7rF2d3KL73xQ3BYynE/0SOReg/BRhzlC0hoWWdoTx/bmxTv
+Za6t2ACGyxSkff4FJJVMiEbt61H5s3Yzm9HXwztZszGIQs9hGIQR1+r7IWfKYGpN
+erdvFiETEmOZ5HFZ9Lxh09dDC+GfQ1kR7f6VkF08gRHFDGo9ih0BsqWxXADjiJfk
+ommkmHF9qrISxhO13nR0ub1zo+wB1QSmObQ2qs89LWvEBa02cpihqnAq3wzPOj11
+OKkctfo/w+4YXwEj9KFdqMBLcCkqrsUohPnwf5BUeQeLWGl7Z/oUnqiA0jQRXzd6
+yHcLZ1Oa2XXY0QIDAQABo4IDpTCCA6EwHQYDVR0OBBYEFO3dVmWvVTnpbS3e3/YM
+PP8YHsl5MDAGA1UdEQQpMCeCJWxuY3MtY3JpbWVwb3J0YWwtbm90aWZ5LXVhdC5s
+bmNzLmhtY3MwHwYDVR0jBBgwFoAUm8b4VBKANon5IGcY0V3hOP4gqlkwggEkBgNV
+HR8EggEbMIIBFzCCAROgggEPoIIBC4aBxGxkYXA6Ly8vQ049RVVDUlNBRU5UQ0Ew
+MDEoMSksQ049RVVDVzQxNzFTQ0EwMDEsQ049Q0RQLENOPVB1YmxpYyUyMEtleSUy
+MFNlcnZpY2VzLENOPVNlcnZpY2VzLENOPUNvbmZpZ3VyYXRpb24sREM9bGV4bmxl
+MSxEQz1pbnRlcm5hbD9jZXJ0aWZpY2F0ZVJldm9jYXRpb25MaXN0P2Jhc2U/b2Jq
+ZWN0Q2xhc3M9Y1JMRGlzdHJpYnV0aW9uUG9pbnSGQmh0dHA6Ly9wa2ljcmxhaWEu
+bGV4bmxlMS5pbnRlcm5hbC9DZXJ0RW5yb2xsL0VVQ1JTQUVOVENBMDAxKDEpLmNy
+bDCCAWsGCCsGAQUFBwEBBIIBXTCCAVkwgbIGCCsGAQUFBzAChoGlbGRhcDovLy9D

```

Comment:
Do w want to keep them here or move into a KV?

I did do similar with APIM APPGW some time ago - https://github.com/hmcts/terraform-module-apim-application-gateway/blob/cee7c19f1edacd1af27494be8cf48fd105c0f4f5/main.tf#L231